### ISSUE TITLE
ContractRequestMessage 'target' in negotiation process required or redundant?

### ISSUE THREAD

**Issue by matgnt** [International-Data-Spaces-Association/ids-specification#90]
In the **catalog** process, the profile explicitly says:
```
Each ODRL Offer must be unique to a dataset since the target of the offer is derived from its enclosing context.
Each ODRL Offer must NOT include an explicit target attribute.
```
https://github.com/International-Data-Spaces-Association/ids-specification/blob/main/catalog/catalog.protocol.md#5-dcat-and-odrl-profiles

How is this during the **negotiation** process?
The example message for `ContractRequestMessage` shows both, a `"dspace:dataSet": "urn:uuid:3dd1add8-4d2d-569e-d634-8394a8836a88"` and a `dspace:offer -> odrl:target` which is the same and probably is always the same, right?
https://github.com/International-Data-Spaces-Association/ids-specification/blob/main/negotiation/message/contract-request-message.json

---
Matthias Binzer
Robert Bosch GmbH

**Comment by matgnt** [comment]
ok, the text also mentions this:
```
The dataset id is not technically required but included to avoid an error where the offer is associated with a different data set.
```

**Comment by matgnt** [comment]
Closing the issue. Resolved with #195
